### DISCUSSION TITLE
Official Discord for PocketBase

### DISCUSSION THREAD

**Discussion by yukathe1st** [pocketbase/pocketbase#5753] [Q&A] [answered]
I know this isn’t the first post on this topic, but an official Discord for PocketBase would be wonderful. Right now, there are multiple unofficial (small) servers, which are not very well-organized so they are not really helpful. @ganigeorgiev, I can totally understand that you don't want to spend unnecessary time on community support. However, Discord is a great platform for community-based support, and if the community can’t help, people are still free to open a Discussion or Issue here. My idea is for the official Discord to be primarily community-run. You wouldn’t need to post there unless you wanted to :). The community would handle most things (like me for example).

**Comment by ganigeorgiev** [comment, marked as answer]
No, I'm not interested in maintaining or promoting another support channel.

**Reply by yukathe1st** [reply]
Ok :(
